src/Logging/TeamCity.php

```diff
@@ -89,11 +153,52 @@ public function startTestSuite(TestSuite $suite): void
         ]);
     }
 
+    /**
+     * Verify that the given test suite is a valid Pest suite.
+     *
+     * @param TestSuite<Test> $suite
+     */
+    private static function isPestTestSuite(TestSuite $suite): bool
+    {
+        return strncmp($suite->getName(), 'P\\', strlen('P\\')) === 0;
+    }
+
+    /**
+     * @param array<string, string|int> $params
+     */
+    private function printEvent(string $eventName, array $params = []): void
+    {
+        $this->write("\n##teamcity[{$eventName}");
```

Comment:
```suggestion
        $this->write("##teamcity[{$eventName}");
```

Actually we should remove this first `\n`, we always append it, so having it both places is not needed. 

I also think we need to do this because phpstorm has the method called `serviceMessageHasNewLinePrefix` which is scheduled for removal in 2021.3.